cos.adoc

```diff
@@ -96,6 +102,12 @@ Preferences page will not display in users mailboxes: they therefore cannot
 modify the default configuration for features that are set up for their
 accounts.
 
+ifdef::z9[]
+[IMPORTANT]
+User can change his Preferences in {modern} under Settings. Currently, disabling Preferences does not have any impact on {modern}.
```

Comment:
Gender-neutral
Users can change their Preferences in {modern} under Settings. Currently, disabling *Preferences* does not have any impact on {modern}.

Reply:
AND... one sentence per line, one line per sentence.